network/802.11/hccap.ksy

```diff
@@ -0,0 +1,57 @@
+meta:
+  id: hccap
+  title: Hashcat capture old format
+  license: Unlicense
+  file-extension: hccap
+  application:
+    - Hashcat
+    - aircrack-ng
+  endian: le
+  encoding: utf-8
+  imports:
+    - /network/mac_address
+    - /network/802.11/eapol_key
+    - /network/802.11/wpa2_nonce
+doc: |
+  Native format of Hashcat password "recovery" utility
+  A sample of file for testing can be downloaded from https://web.archive.org/web/20150220013635if_/http://hashcat.net:80/misc/example_hashes/hashcat.hccap
+doc-ref: https://hashcat.net/wiki/doku.php?id=hccap
+seq:
+  - id: records
+    type: hccap
```

Comment:
This is literally a very bad idea to be that ambiguous. Why not use `hccap_record` or something for the individual record type, why make a clash here?

Reply:
OK.